app/controlplane/api/controlplane/v1/conf.proto

```diff
@@ -22,7 +22,7 @@ import "controlplane/v1/response_messages.proto";
 import "credentials/v1/config.proto";
 import "google/protobuf/duration.proto";
 
-option go_package = "github.com/chainloop-dev/chainloop/app/controlplane/internal/conf/controlplane/config/v1;conf";
+option go_package = "github.com/chainloop-dev/chainloop/app/controlplane/api/controlplane/v1;v1";
 
 message Bootstrap {
```

Comment:
I am not convinced about moving the whole file, the reason been that `Bootstrap` inside is specifically about this app.

What we can do is to move the required sub-structs (like we did for example with credentials).

Reply:
Ok, I'll check what are those required substructs.

Reply:
I've finally moved only OnboardingSpec message, which is set as a dependency for some use cases.